app/controllers/top_controller.rb

```diff
@@ -1,5 +1,5 @@
 class TopController < ApplicationController
   def index
-    @prototypes = Prototype.includes(:prototype_images).order('likes_count DESC').page(params[:page]).per(8)
+    @prototypes = Prototype.includes(:prototype_images).order('likes_count DESC').page(params[:page]).per(4)
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.